spec/models/validations/household_validations_spec.rb

```diff
@@ -674,18 +674,43 @@
   end
 
   describe "housing needs validations" do
-    it "is invalid when a combination of housingneeds == 1 (yes) && housingneeds_type == 3 (none of listed) && housingneeds_other == 0 (no)" do
-      record.housingneeds = 1
-      record.housingneeds_type = 3
-      record.housingneeds_other = 0
+    context "with housingneeds == 2 (yes) && housingneeds_type == 3" do
+      before do
+        record.housingneeds = 1
```

Comment:
This doesn't match the description - I assume that's just a typo?

Reply:
ah thanks 🤦